__tests__/app.test.js

```diff
@@ -34,4 +35,16 @@ describe('GET/api', () => {
         }
       });
   });
+  test('GET:200 compares original json file to be equal to the data sent back from api', () => {
+    const originalEndpointDetails = fs.readFileSync(
+      './endpoints.json',
+      'utf-8'
+    );
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then((res) => {
+        expect(res.body.details).toEqual(JSON.parse(originalEndpointDetails));
+      });
+  });
```

Comment:
This works but just as a reminder you can require in .json files and access the objects within directly without needing to use the fs library.